appengine-java21/bigtable/src/main/webapp/WEB-INF/logging.properties

```diff
@@ -4,9 +4,9 @@
 # To use this configuration, copy it into your application's WEB-INF
 # folder and add the following to your appengine-web.xml:
 #
-# <system-properties>
+#  <system-properties>
```

Comment:
this looks like a change that wasn't meant to happen

Reply:
Fixed